src/layoutparser/models/base_catalog.py

```diff
@@ -10,7 +10,7 @@ class DropboxHandler(HTTPURLHandler):
     """
 
     def _get_supported_prefixes(self):
-        return ["https://www.dropbox.com"]
+        return ["https://www.dropbox.com","https://paddle-model-ecology.bj.bcebos.com"]
```

Comment:
I don't think this is a good idea -- probably it's better to create something like `class PaddleModelURLHandler(HTTPURLHandler):` in the paddle model `catalog.py` file for this purpose.